src/CoreWCF.Primitives/src/CoreWCF/Channels/DefaultAuthorizationCapabilities.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace CoreWCF.Channels;
+
+public class DefaultAuthorizationCapabilities : IAuthorizationCapabilities
+{
```

Comment:
Do we want this class public? The general pattern is for the interface to be public, and the default implementation to be internal. Take a look at ISecurityCapabilities and SecurityCapabilities as an example. If it's internal, you don't need a constructor, a setter will work.

Reply:
indeed i renamed the class too to ensure I follow the general pattern you mentioned https://github.com/CoreWCF/CoreWCF/pull/850/commits/830320d1af7dd6c019176c06aab0a7cc345fefe4